### PR TITLE
auto build docker for Actions

### PR DESCRIPTION
**Pull request by twocolors**
this Actios for github for auto build docker, support platforms: linux/amd64, linux/arm64, linux/arm/v7, linux/arm/v6 and node 14,16,18 

llast tags for nodejs 16

### CODE REVIEW

**Comment by twocolors** [comment]
more info
https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action
https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

**Comment by gfwilliams** [comment]
Have you tried this and it actually works ok? Because I'd have thought that running EspruinoHub in docker was a bit of a nightmare given how it needs access to system Bluetooth?

**Comment by twocolors** [comment]
```
dietpi@holyland:~$ cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
dietpi@holyland:~$ uname -a
Linux holyland 5.15.76-v8+ #1597 SMP PREEMPT Fri Nov 4 12:16:41 GMT 2022 aarch64 GNU/Linux
dietpi@holyland:~$ sudo docker-compose logs -f espruinohub --tail=150
espruinohub  | [Config] Config /data/config.json loaded
espruinohub  | [HTTPProxy] config.http_proxy=false, not enabling Bleno/Proxy
espruinohub  | [MQTT] Connecting...
espruinohub  | [HTTP] No http_port in config. Not enabling web server
espruinohub  | [History] history_path value is empty, thus not providing history.
espruinohub  | [MQTT] Connected
espruinohub  | [Discover] Re-sending presence status of known devices
espruinohub  | [Discover] Noble StateChange: poweredOn
espruinohub  | [Discover] caller if function () {
espruinohub  |     exports.startScan();
espruinohub  |   }
espruinohub  | [Discover] Spawning check-broken interval
espruinohub  | [Discover] Starting Scan
espruinohub  | [Discover] Scanning started.
^Ccanceled
dietpi@holyland:~$ 
```
run on pi4 , example for docker-compose its my config 

Bluetooth for docker only need `--host`

in any case, automatic build and docker image will not hurt

**Comment by twocolors** [comment]
i add article by [Thomas Huffert](https://github.com/espruino/EspruinoHub/pull/103/commits/4118ecb52df6d963f57184e32bf588ccfb1f6f97) , but i use dbus in host and now need `--privileged` , but added as an example, in general, people who run docker understand what they usually do :)

File: README.md [hunk 1]
```diff
@@ -144,13 +144,30 @@ sudo rm -rf ~/EspruinoHub
 Run with Docker
 ---------------
 
-Build on Raspberry Pi Zero:
+More information how work Bluetooth in docker you can read in article "[How to run containerized Bluetooth applications with BlueZ](https://medium.com/omi-uulm/how-to-run-containerized-bluetooth-applications-with-bluez-dced9ab767f6)" by Thomas Huffert
 
-    docker build -t espruino/espruinohub:armhf https://github.com/espruino/EspruinoHub.git
```
Comment (gfwilliams) [inline README.md:149]: Is it worth leaving this line in just in case someone wants to build it themselves at some point?

**Comment by gfwilliams** [comment]
Thanks for adding the privileged bit - I think that would help out a lot of new users wondering why it didn't pick up any bluetooth devices :)

**Comment by twocolors** [comment]
update doc for arch and build

**Comment by gfwilliams** [comment]
That's great - thanks!

**Pull request merged**